$NetBSD: patch-doc_examples_getredirect_c,v 1.2 2016/02/01 13:15:27 mef Exp $

Simple typo in examle code, but this needs make test to pass
(when PKGSRC_RUN_TEST= yes).
I've reported this to upstream.

--- docs/examples/getredirect.c~	2016-01-15 00:37:49.000000000 +0900
+++ docs/examples/getredirect.c	2016-02-01 21:52:01.000000000 +0900
@@ -48,7 +48,7 @@ int main(void)
     else {
       res = curl_easy_getinfo(curl, CURLINFO_RESPONSE_CODE, &response_code);
       if((res == CURLE_OK) &&
-         ((code / 100) != 3)) {
+         ((response_code / 100) != 3)) {
         /* a redirect implies a 3xx response code */
         fprintf(stderr, "Not a redirect.\n");
       }
